docs/source/blueprints/BP003.rst

```diff
@@ -0,0 +1,366 @@
+Task #1 having started).
+
+Because of that, Task #2 is now considered ``FINISHED``.
+
+And Task #1 is still a ``STARTED`` task.
+
+Final Iteration
+~~~~~~~~~~~~~~~
+
+After several iterations with no status changes, and because of a timeout
+implementation at the job level, it's decided that Task #1 is not to be waited
+on.
+
+The spawner continues to inform that Task #1 is alive (from its PoV), but no
+further status message has been received. Provided the spawner has support for
+that, it may attempt to clean up the task (such as destroying a container or
+killing a process). In the end, it's left with.
+
+Tallying results
+~~~~~~~~~~~~~~~~
+
+The nrunner plugin should be able to provide meaningful results to the Job, and
+consequently, to the user, based on the resulting information on the final
+iteration.
+
+Notice that some information will come, such as the ``PASS`` for the first test,
+will come from the "result" given in a status message from the task itself. Some
+other status, such as the ``INTERRUPTED`` status for the second test will not
+come from a status message received, but from a realization of the actual
+management of the task execution. It's expected to other information will also
+have to be inferred, and "filled in" by the nrunner plugin implementation.
+
+In the end, it's expected that results similar to this would be presented::
+
+    JOB ID     : f59bd40b8ac905864c4558dc02b6177d4f422ca3
+    JOB LOG    : /home/cleber/avocado/job-results/job-2020-05-20T17.58-f59bd40/job.log
+     (1/2) tests.py:Test.test_2: PASS (2.56 s)
+     (2/2) tests.py:Test.test_1: INTERRUPT (900 s)
+    RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 1 | CANCEL 0
+    JOB TIME   : 0.19 s
+    JOB HTML   : /home/cleber/avocado/job-results/job-2020-05-20T17.58-f59bd40/results.html
+
+Notice how Task #2 shows up before Task #1 because it was both started first and
+finished earlier. There may be issues associated with the current UI to deal
+with regarding out of order task status updates.
+
+Backwards Compatibility
+***********************
+
+The compatibility of the resulting Job compatible runner
+implementation with the current runner is to be verified by running
+the same set of "Job API feature tests", but with this runner selected
+instead.
+
+There are no compatibility issues with the previous versions of
+itself, or with the non-Job compatible ``nrun`` implementation.
+
+Security Implications
+*********************
+
+None that we can determine at this point.
+
+How to Teach This
+*****************
+
+The distinctive features that the N(ext) Runner provides should be
+properly documented.
+
+Users should not be required to learn about the N(ext) Runner features
+to use it just as an alternative to the current runner implementation.
+
+Related Issues
+**************
+
+Here's a list of all issues related to this blueprint:
+
+ * https://github.com/avocado-framework/avocado/issues?q=is%3Aopen+is%3Aissue+label%3Anrun2run
```

Comment:
You're right, I'll look for the more relevant ones.

Reply:
Since that we are still learning how to write blueprints, this section could be:

1. The current issues that we have and that will be solved by this BP;
2. The list of all issues that this BP will need to be considered "implemented";
3. Both

What do you think?